config/routes.rb

```diff
@@ -5,10 +5,13 @@
   get '/registration', to: 'users#new'
 
   resources :users, only: [:create]
+  resources :parties, only: [:new]
 
   get '/dashboard', to: 'dashboard#index'
 
-  get '/discover', to: 'movies#index'
+  get '/discover', to: 'movies#discover'
+  get '/movies', to: 'movies#index'
+  get '/movies/:id', to: 'movies#show'
```

Comment:
I think ultimately we do want to figure out how to use api_ref rather than id here, since that's what that data is going to be called when we put in the database